tutorials/distributed-calculator/python/app.py

```diff
@@ -32,4 +30,4 @@ def multiply():
     print(f"Calculating {operand_one} * {operand_two}", flush=True)
     return jsonify(math.ceil(operand_one * operand_two * 100000)/100000)
 
-app.run(port=appPort)
+app.run(host="0.0.0.0",port=appPort)
```

Comment:
Why do we need "0.0.0.0" host ip?

Reply:
@paulyuk - to make the value being used explicit.